app/templates/components/settings/contact-info-section.hbs

```diff
@@ -1,11 +1,11 @@
 <form class="ui form" {{action 'submit' on = 'submit'}}>
   <div class="field">
     <label>Email</label>
-    <input type="text" name="email" placeholder="Email">
+    {{input type="text" name="email" placeholder="Email"}}
```

Comment:
I think you mis-understood ... I'm not asking you to replace placeholder with value. I'm asking you to bind the input to the `email` parameter in the contact-info-section.js component file

```
{{input type='email' name='email' placeholder='john.doe@example.com' value=email}}
```